notebooks/Tutorial_Top-K_Multidimensional_Motif_and_Matches_Discovery.ipynb

```diff
@@ -0,0 +1,827 @@
+{
```

Comment:
<p>Since the first example below focuses on "Tumble Dryer", it may be less confusing to focus on an area where there was significant dishwasher activity (i.e., the motif region)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>